### PR TITLE
Allow whitespace in subject name.

### LINKED ISSUE #18: Support `name` containing whitespace
The current implementation disallows whitespace in the subject `name` because it uses `strings.Fields`. The spec does not disallow whitespace, and spaces in filenames are not uncommon, so it would be best if the implementation allowed that.

To fix, using `strings.Cut` (or `strings.SplitN` or a regexp) would avoid the issue.

### PR DESCRIPTION
**Pull request by ianlewis**
Fixes #18

### CODE REVIEW

**Review by MarkLodato — APPROVED** [review]

**Comment by ianlewis** [comment]
BTW, I don't have commit access so someone has to merge these PRs for me once they are approved or give me access so I can do it myself.

**Comment by MarkLodato** [comment]
Oops! Just gave you access now.

**Pull request merged**
